src/lib/classes/lsp3-universal-profile.spec.ts

```diff
@@ -65,4 +65,32 @@ describe('LSP3UniversalProfile', () => {
       },
     });
   });
+  it.skip('should deploy and set LSP3Profile data', async () => {
+    const myLSPFactory = new LSPFactory(
+      provider,
+      '0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80'
+    );
+
+    const contracts = await myLSPFactory.LSP3UniversalProfile.deploy(
+      {
+        controllingAccounts: ['0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266'],
+        lsp3Profile: lsp3ProfileJson,
+      },
+      {
+        libAddresses: {
+          erc725AccountInit: baseContracts.universalProfile.address,
+          universalReceiverDelegateInit: baseContracts.UniversalReceiverDelegate?.address,
+        },
+      }
+    );
+
+    const ownerAddress = await contracts.ERC725Account.owner();
+    expect(ownerAddress).toEqual('0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266');
```

Comment:
Duplicate value with line 76, you can put this in a variable and reuse same variable at line 76 and 88.
Will also allow to name these addresses which can be useful